Makefile

```diff
@@ -5,7 +5,7 @@ VERSION = $(shell git describe --tags --always)
 # Builds exporter binary
 .PHONY: exporter
 exporter:
-	go build -ldflags="-X 'main.version=$(VERSION)'" -o aerospike-prometheus-exporter .
+	GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags="-X 'main.version=$(VERSION)'" -o aerospike-prometheus-exporter .
```

Comment:
Get default values for `GOOS` and `GOARCH` like you did in the `pkg/Makefile`.

Reply:
What default values? My thought here is that if $(GOOS) and $(GOARCH) are empty then Go will build for whatever the host machine is.